.storybook/RenderPerformanceProfiler.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import isChromatic from 'chromatic/isChromatic';
+import {version} from '../packages/paste-core/core-bundle/package.json';
+
+const trackRenderPerformance = (data) => {
+  const commitSha = process.env.STORYBOOK_GITHUB_SHA ? process.env.STORYBOOK_GITHUB_SHA : 'localdev';
+
+  // TODO: replace with https://paste-data.netlify.app/.netlify/functions/create-component-perf-metric
```

Comment:
Why is this TODO?

Reply:
I need to merge the other PR in the other repo to get that url working. It's running off of a preview url right now.

Reply:
👍🏽